src/utils/MappingUtil.tsx

```diff
@@ -59,3 +59,41 @@ export function addEsriLayersFromVectorSourceStyle(style: any, map: mapboxgl.Map
   });
 }
 }
+
+function getBoundingBox(coordinates: any[]) {
+  let bbox: any = {
+    xMin: undefined,
+    xMax: undefined,
+    yMin: undefined,
+    yMax: undefined,
+  }
+
+  coordinates.forEach((c)=>{
+    const longitude = c[0];
+    const latitude = c[1];
+    bbox.xMin = bbox.xMin < longitude ? bbox.xMin : longitude;
```

Comment:
Nit: could use Math.max and Math.min instead

Reply:
![image](https://user-images.githubusercontent.com/22464147/123304180-de73f600-d4db-11eb-9640-bd8c5314d5d5.png)